lib/github-glue.ts

```diff
@@ -70,7 +70,7 @@ export class GitHubGlue {
             throw new Error(`Could not find ${gitGitCommit}: '${output}'`);
         }
         const [, short, completedAt] = match;
-        const url = `https://github.com/git/git/commit/${gitGitCommit}`;
+        const url = `https://github.com/${baseOwner}/${this.repo}/commit/${gitGitCommit}`;
```

Comment:
The fork could have been renamed, though. See e.g. https://github.com/git-for-windows/msys2-runtime/, which was forked from https://github.com/Alexpux/Cygwin.

Reply:
I merged the PR anyway because the "base repo has a different name than fork" problem is not one we have to face _now_.